src/styles/nav.scss

```diff
@@ -27,6 +27,16 @@
   z-index: 100;
 }
 
+.side__social-links {
+  position: absolute;
```

Comment:
can you make this so that it's not sticky? because there are also links at the bottom of the page (same icons)